xen/console: Set the default log level to INFO for release builds

With the default log level as it currently is, the messages for applying
and reverting live patches do not get logged as was originally intended.
By setting the default log level to include INFO, these messages are
printed to the console.

While this change does increase the boot verbosity a little, it doesn't
appear to cause additional log messages to be displayed during normal
operation.

Signed-off-by: Ross Lagerwall <ross.lagerwall@citrix.com>

diff --git a/xen/drivers/char/console.c b/xen/drivers/char/console.c
index f4f6141..3f846a6 100644
--- a/xen/drivers/char/console.c
+++ b/xen/drivers/char/console.c
@@ -92,8 +92,8 @@ static DEFINE_SPINLOCK(console_lock);
  * the lower threshold equal to the upper.
  */
 #ifdef NDEBUG
-#define XENLOG_UPPER_THRESHOLD       2 /* Do not print INFO and DEBUG  */
-#define XENLOG_LOWER_THRESHOLD       2 /* Always print ERR and WARNING */
+#define XENLOG_UPPER_THRESHOLD       3 /* Do not print DEBUG  */
+#define XENLOG_LOWER_THRESHOLD       3 /* Always print INFO, ERR and WARNING */
 #define XENLOG_GUEST_UPPER_THRESHOLD 2 /* Do not print INFO and DEBUG  */
 #define XENLOG_GUEST_LOWER_THRESHOLD 0 /* Rate-limit ERR and WARNING   */
 #else
